### PR TITLE
Update cifar10_tutorial.py

### PR DESCRIPTION
**Pull request by junecv**
fixed running loss per iteration

### CODE REVIEW

File: beginner_source/blitz/cifar10_tutorial.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def forward(self, x):
         # print statistics
         running_loss += loss.item()
         if i % 2000 == 1999:    # print every 2000 mini-batches
-            print(f'[{epoch + 1}, {i + 1:5d}] loss: {running_loss / 2000:.3f}')
+            print(f'[{epoch + 1}, {i + 1:5d}] loss: {running_loss / (i + 1):.3f}')
```
Comment (malfet) [inline beginner_source/blitz/cifar10_tutorial.py:185]: This feels wrong unless you delete the next line that resets running_loss

**Review by malfet — CHANGES_REQUESTED** [review]
`running_loss` is accumulated over 2000 iterations, should it should be divided by 2000, isn't it?

**Comment by junecv** [comment]
You're right. I wasn't aware of the reset line in the original tutorial.

**Comment by junecv** [comment]
Thank you for your time!
